WebPortal/Components/FileGridComponents/FileGridFolder.razor

```diff
@@ -34,8 +34,8 @@
         </div>
     </div>
     <div class="grid-item">@Folder.lastmodifiedts.ToShortDateString()</div>
-    <div class="grid-item">@MSGraphService.GetUserName(Folder.lastmodifiedby)</div>
-    <div class="grid-item">@MSGraphService.GetUserName(Folder.createdby)</div>
+    <div class="grid-item">@(async () => await MSGraphService.GetUserName(Folder.lastmodifiedby))</div>
```

Comment:
I think this will not get the value but show the something else.

Reply:
You are right. I will fix that